macroslib/tests/expectations/cenum.cpp

```diff
@@ -0,0 +1,21 @@
+// Test for minimal C Enum types support.
+"    uint32_t Boo_return_is_enum();";
+
+"    void Boo_param_is_enum(uint32_t e);";
+
+"    uint32_t Boo_param_and_return_are_enum(uint32_t e);";
+
+
+// Important !
+// This new type needs to be named as 'SomeEnumOpaque', 
+// else this conflicts with C++ bindings code (type 'SomeEnum' is already defined there).
+r#"enum SomeEnum {
```

Comment:
In this case more robust test case would be to add the whole file content to test with "pragma once and so on",
to distinguish from C++ header. Because of this test passes even without  new `generate_c_code_for_enum`,
because of C++ header contains the same code.

Reply:
Fixed.